specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2023-08-01-preview/machineLearningServices.json

```diff
@@ -2774,6 +2889,33 @@
         "isLatestOsImageVersion": {
           "type": "boolean",
           "description": "Specifies whether this compute instance is running on the latest operating system image."
+        },
+        "osPatchingStatus": {
+          "readOnly": true,
+          "description": "Metadata about the os patching.",
+          "$ref": "#/definitions/OsPatchingStatus"
+        }
+      }
+    },
+    "OsPatchingStatus": {
+      "type": "object",
+      "description": "Returns metadata about the os patching.",
+      "properties": {
+        "patchStatus": {
+          "type": "string",
+          "description": "The os patching status."
+        },
+        "latestPatchTime": {
```

Comment:
>latestPatchTime [](http://example.com/codeflow?start=9&length=15)

Should this be in datetime format, not string?

Reply:
other property like createdOn, modifiedOn are also defined as string. I'm following the same pattern